src/player.cpp

```diff
@@ -10443,54 +10449,80 @@ bool player::can_wield( const item &it, bool interactive ) const
     return true;
 }
 
-bool player::wield(item* it, bool autodrop)
+bool player::can_unwield( const item& it, bool interactive ) const
```

Comment:
While some of the functions use `interactive` as name of the `bool` argument, it would be clearer if it was `print_msg` or something like that.
That is, unless you intend to make it ask the player what to do on failure or otherwise solve ambiguities within the function.